mods/screwdriver/init.lua

```diff
@@ -19,6 +11,72 @@ screwdriver.rotate_simple = function(pos, node, user, mode, new_param2)
 	end
 end
 
+-- For attached wallmounted nodes: returns true if rotation is valid
+local function check_attached_node(pos, rotation)
```

Comment:
This is a simplified version of the engine method  https://github.com/minetest/minetest/blob/master/builtin/game/falling.lua#L148.

Maybe a common engine function `is_rotation_acceptable` would be more general.